chat-demo-android/build.gradle

```diff
@@ -9,23 +9,23 @@ def ACCESS_TOKEN_SERVICE_URL = project.hasProperty('ACCESS_TOKEN_SERVICE_URL') ?
 
 dependencies {
     implementation "androidx.appcompat:appcompat:$androidXVersion"
-    implementation 'com.google.android.material:material:1.0.0'
+    implementation 'com.google.android.material:material:1.2.1'
 
     // FCM
     // Firebase now requires the app gradle file to explicitly list
     // `com.google.firebase:firebase-core` as a dependency for Firebase services to work as expected.
-    implementation 'com.google.firebase:firebase-core:17.4.0'
-    implementation 'com.google.firebase:firebase-iid:20.1.6'
-    implementation 'com.google.firebase:firebase-messaging:20.1.6'
+    implementation 'com.google.firebase:firebase-core:17.5.1'
+    implementation 'com.google.firebase:firebase-iid:20.3.0'
+    implementation 'com.google.firebase:firebase-messaging:20.3.0'
 
     // Crashlytics
-    implementation 'com.google.firebase:firebase-crashlytics:17.0.0'
-    implementation 'com.google.firebase:firebase-crashlytics-ndk:17.0.0'
+    implementation 'com.google.firebase:firebase-crashlytics:17.2.2'
+    implementation 'com.google.firebase:firebase-crashlytics-ndk:17.2.2'
 
     implementation "com.jakewharton:kotterknife:0.1.0-SNAPSHOT"
     implementation "eu.inloop:simplerecycleradapter:0.3.4"
 
-    implementation 'com.twilio:chat-android-with-symbols:5.0.1'
+    implementation 'com.twilio:chat-android-with-symbols:6.1.1'
```

Comment:
It was [breaking change](https://www.twilio.com/docs/chat/android/changelog#breaking-changes) introduced in `6.0.0` 
> Now `Channel.getUnconsumedMessagesCount()` returns null instead of 0 if the message consumption horizon was not set.
`Messages.setNoMessagesConsumedWithResult()` always returns null instead of 0 (because it resets the consumption horizon).

So we have to fix all possible `NullPointerException`s that now could happen because of that change

Reply:
Oke

Reply:
Do you mean callback to the listener?

Reply:
Yes, exactly

Reply:
Done